internal/core/layout.rs

```diff
@@ -191,14 +191,13 @@ mod grid_internal {
 
             //let to_distribute = size_without_spacing - (size_cannot_grow + current_size);
             let to_distribute =
-                A::to_distribute(size_without_spacing, size_cannot_grow + current_size) as f32;
+                A::to_distribute(size_without_spacing, size_cannot_grow + current_size);
             if to_distribute <= 0. || max_grow <= 0. {
                 return Some(());
             }
 
             let grow = if total_stretch <= 0. {
-                to_distribute
-                    / (data.iter().filter(|it| A::can_grow(it) > 0 as _).count() as Coord) as f32
+                to_distribute / (data.iter().filter(|it| A::can_grow(it) > 0 as _).count() as Coord)
```

Comment:
the thing is that Coord can be f32 or i32 depending on the compile option, so lots of conversion are actually necessary.